src/python/turicreate/toolkits/object_detector/_tf_model_architecture.py

```diff
@@ -0,0 +1,390 @@
+# -*- coding: utf-8 -*-
+# Copyright © 2017 Apple Inc. All rights reserved.
+#
+# Use of this source code is governed by a BSD-3-clause license that can
+# be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+
+from __future__ import print_function as _
+from __future__ import division as _
+from __future__ import absolute_import as _
+import tensorflow as _tf
+import numpy as _np
+
+
+class ODTensorFlowModel(object):
+
+    def __init__(self, input_image_shape, batch_size, output_size, init_weights, config, is_train=True):
+        _tf.compat.v1.get_default_graph()
```

Comment:
What does this do?

Reply:
This gets the default graph and makes the updates to the graph that has been called. In our case, since we only have one graph, this can be safely removed since everything is added to the default graph.